nix/sources.nix

```diff
@@ -0,0 +1,138 @@
+# This file has been generated by Niv.
```

Comment:
Is it standard practice to commit generated files? Can't `Niv` regenerate this on the fly, or are there some custom things that it cannot infer alone?

Reply:
basically the `niv` workflow is orthogonal to the build. you need niv only to init the adapter file here and to update and add dependencies in `sources.json`. you don't need it for the build, and it would be problematic if you did, since it would need to be installed globally, making it impossible to build without extra instructions.